themes/djangoday/layouts/shortcodes/talklist.html

```diff
@@ -20,7 +20,7 @@ <h2>
   <img src="{{ $new.Permalink }}" alt="Speaker" class="img-speaker float-right" />
 
   <h2 class="no-link-styling">
-  <!--<span class="talktime">{{ dateFormat "15:04" .Params.talk_date }}</span>-->
+  <span class="talktime">{{ dateFormat "15:04" .Params.talk_date }}</span>
```

Comment:
Can we comment this one out again? That way, we can maintain the time schedule without announcing it :)

Reply:
Makes sense @benjaoming thanks, pushed. You can merge if it looks good to you.